models/food.js

```diff
@@ -16,5 +16,20 @@ module.exports = (sequelize, DataTypes) => {
       })
     })
   }
+
+  Food.updateItem = function(req) {
+    return new Promise(function(resolve, reject) {
+      Food.update(
```

Comment:
This might not cover our requirements for the name and calories being required in order to update the record.

Reply:
Added the check to the function